docs/make.jl

```diff
@@ -9,6 +9,7 @@ makedocs(;
     repo="https://github.com/invenia/BlockDiagonals.jl/blob/{commit}{path}#L{line}",
     sitename="BlockDiagonals.jl",
     authors="Invenia Technical Computing",
+    strict=true,
```

Comment:
Needed so tht doc's building errors if they doctests don't pass.
Because we aren't testing that in the tests anymore